Website clone

rehanvrana opened the pull request:
Added Remaining components of Home page and completed design of home page.

rehanvrana committed:
Added Team Component

Add Responsive Team and Tabs Component inside the task

rehanvrana committed:
Added Footer

Added all Homepage Component and Make Responsive Homepage

JaydeepParmar-iView commented on src/components/BlogComponent/Blogs.tsx:
```diff
@@ -0,0 +1,66 @@
+      "In this tutorial, we will discuss the Hibernate Query Language. HQL is an object-oriented query language. Hibernate Query",
+    ],
+  ];
+
```
Instead of using Array of Arrays use Array of Objects and give proper keys names in object. For example - [{name:'ABC', date:'11-12-2014'},{name:'ABC', date:'11-12-2014'}].

rehanvrana committed:
change data and Add animation on components

change array of array to array of objects

rehanvrana committed:
Scroll animation and coutup animation Added

Scroll animation is added in slider and service component and count up animation added in experience component
